Lib/test/test_urllib2_localnet.py

```diff
@@ -306,7 +306,7 @@ def test_basic_auth_success(self):
         try:
             self.assertTrue(urllib.request.urlopen(self.server_url))
         except urllib.error.HTTPError:
-            self.fail("Basic auth failed for the url: %s", self.server_url)
+            self.fail(f"Basic auth failed for the url: {self.server_url}")
```

Comment:
Using %-formatting rather than an f-string would make backporting this much easier, since this is to be backported to 2.7.